atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/TargetedSweepMetrics.java

```diff
@@ -172,6 +176,7 @@ private static String getTagForStrategy(SweeperStrategy strategy) {
         private final Gauge<Long> millisSinceLastSwept;
         private final SweepOutcomeMetrics outcomeMetrics;
         private final SlidingWindowMeanGauge batchSizeMean;
+        private final CurrentValueMetric<Long> sweepDelay;
```

Comment:
nit: as above, rename to `sweepDelayMetric`

Reply:
ah now I understand, will do 👍

Reply:
you didn't rename this